src/rabbitmq/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM rabbitmq:latest
```

Let's name stages of building the Dockerfiles. It can be useful in the future when(if) Dockerfile changes and expandes.
```suggestion
FROM rabbitmq:latest as build-stage
```